packages/gatsby-theme-docs/src/data/top-menu.yaml

```diff
@@ -24,6 +24,6 @@
 - menu-title: Merchant Center
   items:
     - label: Documentation
-      href: /../merchant-center
```

Comment:
just for demo, it has to be rolled back

Reply:
You can adjust the megamenu content in the kit repository to, for example, have an actual structure like
 * Documentation
   *  Docs Kit Docs
 * Test Sites
    * API Docs Smoke Test
    * Docs Smoke Test
    * Site Template
  
It would fix an old long-standing issue with the test and docs site - it's completely confusing that they have the actual commercetools docs megamenu. 

The Top Bar would say  "Documentation -> Docs Kit Docs"  or "Test Sites -> API Docs Smoke Test" then.

Reply:
(I know, it opens a rabbithole on how the megamenu content is actually to be distributed into the actual content repositories.  But as you can see, the one here is out of date anyways.

Reply:
don't feel like jumping into this rabbithole, at least in the scope of this ticket. I've changed the `menu-title` so that we don't beed to make any string comparison to display the Title in the top bar. If we want to address other issues related to the menu configuration I'd rather create a separate ticket properly groomed with well defined requirements.

Reply:
@nkuehn Do we already have a ticket to address the "distribution" of the production top menu data?